enterprise/api/v1/serializers.py

```diff
@@ -97,3 +97,27 @@ class Meta:
     user = UserSerializer()
     enterprise_customer = EnterpriseCustomerSerializer()
     data_sharing_consent = UserDataSharingConsentAuditSerializer(many=True)
+
+
+class EnterpriseCustomerUserEntitlementSerializer(serializers.Serializer):
```

Comment:
We've been including the Read/Write capabilities of these classes in their names -- at least we've done this for some others -- so for example this might be `EnterpriseCustomerUserEntitlementReadOnlySerializer`